cloudinit/config/cc_rightscale_userdata.py

```diff
@@ -44,6 +44,7 @@
     "distros": [ALL_DISTROS],
     "frequency": PER_INSTANCE,
     "examples": [],
+    "activate_by_schema_keys": [],
```

Comment:
So, in reviewing this. This module does not work at all currently. It calls `[cloud.get_userdata_raw()` which is a method that doesn't exist and then drops into a very generic, unhelpful exception message.](https://github.com/canonical/cloud-init/blob/main/cloudinit/config/cc_rightscale_userdata.py#L73)   The reason this errors always is
```
2022-07-21 21:10:30,402 - cc_rightscale_userdata.py[DEBUG]: Failed to get raw userdata in module cc_rightscale_userdata 'Cloud' object has no attribute 'get_userdata_raw'
```

So,  this looks like it broke in commit 32362c0ac63f10d7a33e3b95dd91a544a1cbdf5 . I've contacted support at Flexera (which now owns RightScale) to ensure they have no use for this module anymore so that we can remove it instead of fixing it if unneeded. That'd be a separate PR if they concur their products don't use cloud-init.

Reply:
Thanks for pointing this out. I did diagonally read the modules and did not notice it.